pages/content/amp-dev/500.html

```diff
@@ -0,0 +1,25 @@
+---
+$title: 500
+$view: /views/custom.j2
+---
+{% do doc.styles.addCssFile('css/components/templates/error.css') %}
+{% do doc.styles.addCssFile('/css/components/molecules/jumbo.css') %}
+{% do doc.styles.addCssFile('/css/components/molecules/biggy.css') %}
+
+<main class="ap--main ap-t-error">
+  <section class="ap--container">
+
+    <div class="ap-m-jumbo">
+      <div class="ap-a-ico">
+        {% do doc.icons.useIcon('/icons/bolt-broken-solid.svg') %}
+        <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#bolt-broken-solid"></use></svg>
+      </div>
+      <h1 class="ap-m-biggy-headline">500</h1>
+      <h2 class="ap-m-jumbo-subline">{{ _('Uh oh! That\'s an error.') }}</h2>
+      <p class="ap-m-jumbo-copy">{{ _('Something went terribly wrong. Why don\'t you browse around?') }}</p>
+
+      {% do doc.styles.addCssFile('/css/components/atoms/button.css', -1) %}
+      <a href="https://events.withgoogle.com/amp-conf-2019/registrations/new/" class="ap-a-btn">{{ _('Back to home') }}</a>
```

Comment:
That's already wrong in 404.html, sorry could have given you a hint:

```suggestion
      <a href="{{ g.doc('/content/amp-dev/index.html', locale=doc.locale) }}" class="ap-a-btn">{{ _('Back to home') }}</a>
```

Reply:
fixed